dcp_prototype/backend/wrangling/README.md

```diff
@@ -9,15 +9,48 @@ transforms them into valid DCP 2.0 metadata objects, and inputs them into
 the DCP 2.0 ledger.  Specify the location to the DCP 1.0 files, and the number
 of threads to use for accessing S3.
 
+The location can be one of the following:
+ * an s3 bucket containing metadata files and individual object
+ * an s3 object which contains all dcp1 metadata as a tar.gz file 
+ * a local tar.gz file
+ * a local directory containing individual files
+ 
+It is much much faster to use a tar.gz file.
+When using a .tar.gz file it is no longer necessary to run with multiple threads.
+
 Example use:
 ```
-    python3 migrations/scripts/migrate_dcp_one_to_dcp_two.py -i <s3dir> --threads <nthreads>
+    python3 migrations/scripts/migrate_dcp_one_to_dcp_two.py -i <source> --threads <nthreads>
 ```
 
 Concrete example:
 ```
-    python3 migrations/scripts/migrate_dcp_one_to_dcp_two.py \
-       -i  s3://dcp-ledger-bucket-dev/single_cell_transcriptome_analysis_of_human_pancreas_metadata/
-       --threads 20
+    export BUCKET=s3://hca-dcp-one-backup-data/dcp-one-copy
+    ecport PROJECT=Single-cell-transcriptome-analysis-of-human-pancreas
+    python3 migrations/scripts/migrate_dcp_one_to_dcp_two.py -i $BUCKET/$PROJECT.tar.gz 
+```
+
+### migrations/scripts/metadata_aggregator.py
+
+Script to create a tar.gz file from the metadata contained in an S3 bucket.
+It first downloads all the files locally, creates the .tar.gz file, then uploads them back to s3.
```

Comment:
This could potentially be a lot downloaded locally right?

Reply:
The point of this script is to download all the files once, then upload a .tar.gz files.  From then onwards you only need to deal with the single .tar.gz file.
I have already done this for all the projects.  So, take a project like tabular-muris, which takes about an hour to download all the 500K small json files, you can now download the .tar.gz in less than one second.